examples/gameserverallocation.yaml

```diff
@@ -85,34 +85,34 @@ spec:
       mode: deathmatch
     annotations:
       map: garden22
-  # [Stage: Alpha]
```

Comment:
This one doesn't look right to me. It's indenting it under .metadata.annotations (visually) but I don't think that's where the structure is. Can you disable yammlint for this block?

Reply:
I have rolled back the modifications in this file and added it to the yamllint configuration to be ignored.

Reply:
I think it's better for pointwise suppressions like this to use comments to suppress it: https://yamllint.readthedocs.io/en/stable/disable_with_comments.html

Reply:
The following options to disable yamllint are working fine, I chose the first one, please let me know what you think..

  # yamllint disable rule:comments-indentation rule:line-length
and 
  # yamllint disable

Reply:
LGTM